### PR TITLE
YaTelemetry Integration

### PR DESCRIPTION
**Pull request by felipevb**
The following MR performs the following:
 - Replaces ZMQ approach to Downrupt and reimplements with yaTelemetry integration
 - Removes ZMQ / serde / serde_json dependencies due to this change
 - General code cleanup and static dependency versions

### CODE REVIEW

File: src/mem/periph/downrupt.rs [hunk 1]
```diff
@@ -1,15 +1,38 @@
...
+            Ok(mut xa) => {
+                loop {
+                    let msg = match rx.recv() {
+                        Ok(x) => x,
+                        _ => {
+                            break;
+                        }
+                    };
+
+                    match xa.write(&msg) {
```
Comment (dyerti) [inline src/mem/periph/downrupt.rs:25]: You can lose data here. `write` is fallible and also doesn’t guarantee complete transmission of bytes.

Comment (felipevb) [reply src/mem/periph/downrupt.rs:25]: Good catch. I know this and the dsky follow the same approach so they probably have the same issue. Initial thought is to capture the results and break out of the loop if any errors occur. 

Curious on the guarantee of the complete transmission of bytes. My assumption was either it sends all or errors out at any point in between 0 and n. My assumption is the yaTelemetry will handle the parsing if channels are disconnected and reconnected.

Comment (dyerti) [reply src/mem/periph/downrupt.rs:25]: You are looking for `write_all` which behaves that way.

Comment (felipevb) [reply src/mem/periph/downrupt.rs:25]: I will create a new issue to update my `write` to `write_all`. Thanks for letting me know.

**Pull request merged**